cv/qc-cv.obo

```diff
@@ -417,6 +417,1146 @@ is_a: QC:4000024 ! MS2 metric
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000074 ! Sequence coverages
+has_column: QC:4000075 ! Protein accessions
+has_column: QC:4000076 ! Length values
+has_optional_column: QC:4000077 ! Target or decoy designation
+
+[Term]
+id: QC:4000084
+name: Sampling rates
+definition: "The sampling rates of identified peptides and their respective frequency (accumulated by the sampling rate, i.e. how often peptides were sampled n times and how many peptides were resampled at that rate. Optionally, which peptides were sampled at that rate, space separated)" [PSI:QC]
+comment: The sampling rate may give insights to peptide separation, dynamic exclusion settings and gradient efficiency.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000078 ! Sample rates
+has_column: QC:4000079 ! Counts
+has_optional_column: QC:4000080 ! Peptide sequences
+
+[Term]
+id: QC:40000
+name: Explained precursor intensity first quarter
+def: "Fraction of identified MS2 in the first quarter of precursor intensity." [PSI:QC]
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4A" NARROW []
+
+[Term]
+id: QC:40000
+name: Explained precursor intensity second quarter
+def: "Fraction of identified MS2 in the second quarter of precursor intensity." [PSI:QC]
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4B" NARROW []
+
+[Term]
+id: QC:40000
+name: Explained precursor intensity third quarter
+def: "Fraction of identified MS2 in the third quarter of precursor intensity." [PSI:QC]
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4C" NARROW []
+
+[Term]
+id: QC:40000
+name: Explained precursor intensity fourth quarter
+def: "Fraction of identified MS2 in the fourth quarter of precursor intensity." [PSI:QC]
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4D" NARROW []
+
+[Term]
+id: QC:40000
+name: Extent of identified precursor intensity
+def: "Ratio of 95th over 5th percentile of precursor intensity for identified peptides" [PSI:QC]
+comment: Can be used to approximate the dynamic range of signal
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+has_units: STATO:0000300 ! dimensionless ratio
+synonym: "MS1-3A" NARROW []
+
+[Term]
+id: QC:40000
+name: Precursor intensity range of identified MS2
```

Comment:
there is `Precursor intensity of identified MS2 Q1, Q2, Q3` and `Precursor intensity of unidentified MS2 Q1, Q2, Q3`, but there is no `Precursor intensity range of unidentified MS2`

Reply:
I didn't add that as it will probably not be that informative; there is going to be unidentified all over the int range.